core/src/stored/append.h

```diff
@@ -21,11 +21,39 @@
 #ifndef BAREOS_STORED_APPEND_H_
 #define BAREOS_STORED_APPEND_H_
 
+
+#include "include/bareos.h"
+#include "lib/bsock.h"
+#include "stored/device_control_record.h"
+#include "stored/record.h"
+
 namespace storagedaemon {
 
+class ProcessedFile {
+ public:
+  ProcessedFile() = default;
+  ~ProcessedFile();
+  ProcessedFile(const ProcessedFile& other);
+  ProcessedFile& operator=(const ProcessedFile& other);
+  ProcessedFile(ProcessedFile&& other);
+  ProcessedFile& operator=(ProcessedFile&& other);
+
+  void SendAttributesToDirector(JobControlRecord* jcr);
+  void Initialize(int32_t index);
+  void AddAttribute(DeviceRecord* record);
+  inline int32_t GetFileIndex() { return fileindex_; }
+  inline std::vector<DeviceRecord> GetAttributes() { return attributes_; }
```

Comment:
When defining the function inside the class' body, the `inline` is implied.
```suggestion
  int32_t GetFileIndex() { return fileindex_; }
  std::vector<DeviceRecord> GetAttributes() { return attributes_; }
```